proto/relay.proto

```diff
@@ -22,33 +22,46 @@ message Entity {
 }
 
 service RelayService {
-  rpc RelayConnect(stream RelayMessage) returns (stream RelayMessage);
+  rpc RelayConnect(stream RequestStream) returns (stream ResponseStream);
```

Comment:
For backwards compat, we could add a new method with this signature and retain the legacy one as well.